src/preact/base-element.js

```diff
@@ -64,13 +64,13 @@ const SHADOW_CONTAINER_ATTRS = dict({
 /** @const {string} */
 const SERVICE_SLOT_NAME = 'i-amphtml-svc';
 
-/** @const {!JsonObject<string, string>} */
+/** @const {JsonObject<string, string>} */
 const SERVICE_SLOT_ATTRS = dict({'name': SERVICE_SLOT_NAME});
 
 /** @const {string} */
 const RENDERED_ATTR = 'i-amphtml-rendered';
 
-/** @const {!JsonObject<string, string>} */
+/** @const {JsonObject<string, string>} */
```

Comment:
JsonObject isn't a generic

Reply:
This PR doesn't address type checking.

Reply:
Oh I didn't realize this wasn't type-checked. Carry on then 👍🏻